packaging/packager_win.bat

```diff
@@ -1,4 +1,4 @@
-@Echo off
+REM @Echo off

```

Comment:
Just one quick comment, why is this commented out? It is not needed then?

Reply:
this takes of the Echo and debugging gets pretty difficult
I suggest we drop this so that we can see exactly what are we doing

Reply:
I think this is a good idea then, don't recall why it was there on the first place. Let's remove it.